### PR TITLE
fix(rating): fix usage of aria-valuetext

### LINKED ISSUE #5571: uib-rating: The WAI-ARIA property aria-valuetext is not supported on elements with WAI-ARIA role button
The issues forum is __NOT__ for support requests. It is for bugs and feature requests only.
Please read https://github.com/angular-ui/bootstrap/blob/master/CONTRIBUTING.md and search
existing issues (both open and closed) prior to opening any new issue and ensure you follow the instructions therein.

### Bug description:
The WAI-ARIA property aria-valuetext is not supported on elements with WAI-ARIA role button.
This is for uib-rating star element, and is related with accessibility. 
https://www.w3.org/TR/wai-aria/states_and_properties#aria-valuetext

<i class="glyphicon ng-scope glyphicon-star" **aria-valuetext**="one" ng-attr-title="{{r.title}}" ng-class="$index < value && (r.stateOn || 'glyphicon-star') || (r.stateOff || 'glyphicon-star-empty')" ng-click="rate($index + 1)" ng-mouseenter="enter($index + 1)" ng-repeat-end="" role="**button**" tabindex="0" title="one"></i>

### Version of Angular, UIBS, and Bootstrap

Angular: 1.4.9

UIBS: 1.2.1

Bootstrap: 3.3.6

### PR DESCRIPTION
**Pull request by wesleycho**
- Fix usage to be on top level element

This fixes #5571.

### CODE REVIEW

File: src/rating/rating.js [hunk 1]
```diff
@@ -8,7 +8,8 @@ angular.module('ui.bootstrap.rating', [])
 })
 
 .controller('UibRatingController', ['$scope', '$attrs', 'uibRatingConfig', function($scope, $attrs, ratingConfig) {
-  var ngModelCtrl = { $setViewValue: angular.noop };
+  var ngModelCtrl = { $setViewValue: angular.noop },
+    self = this;
```
Comment (deeg) [inline src/rating/rating.js]: Not that it really matters, but won't ```this.getTitle``` work in this situation, making self = this not needed?

**Comment by deeg** [comment]
This LGTM

File: src/rating/rating.js (hunk 1 above)
Comment (wesleycho) [inline src/rating/rating.js]: It actually doesn't - I tried that first :(

Comment (deeg) [inline src/rating/rating.js]: Hmmm, do you know why that does not work? Just curious.

Comment (wesleycho) [inline src/rating/rating.js]: Because it is called in the render function, which is called in $scope.rate is my suspicion.

**Pull request closed without merging**